app/helpers/claims_helper.rb

```diff
@@ -61,6 +61,6 @@ def academic_years(year_range)
 
     start_year, end_year = year_range.split("_")
 
-    "September 1 #{start_year} - August 31 #{end_year}"
+    "September 1 #{start_year} to August 31 #{end_year}"
```

Comment:
While we're here, can we switch to using the localization function: `l(date)`? Dates should be `1 September 2013`, not `September 1 2019` according to the design system, and i18n is set up to do that.